src/time_integration/paired_explicit_runge_kutta/methods_PERK2.jl

```diff
@@ -0,0 +1,362 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+# Abstract base type for both single/standalone and multi-level P-ERK time integration schemes
+abstract type PERK end
+# Abstract base type for single/standalone P-ERK time integration schemes
+abstract type PERKSingle <: PERK end
```

Comment:
A bit more verbose names may be nice for people not familiar with all this stuff?